weave/trace/weave_client.py

```diff
@@ -641,11 +651,17 @@ def create_call(
         unbound_op = maybe_unbind_method(op)
         op_def_ref = self._save_op(unbound_op)
 
-        inputs_redacted = redact_sensitive_keys(inputs)
```

Comment:
did you remove redaction?

Reply:
uh, no, now its registered in `parse_obj_functions`